src/at_tools/tools.rs

```diff
@@ -98,6 +99,18 @@ tools:
         description: "The exact name of a function, method, class, type alias. No spaces allowed."
     parameters_required:
       - "symbol"
+
+  - name: "doc_sources"
```

Comment:
I would describe all that with a little bit more details so model would have more clue how to use it properly